.editorconfig

```diff
@@ -3,8 +3,8 @@ root = true
 
 [*]
 charset = utf-8
-indent_style = space
-indent_size = 2
+indent_style = tab
+indent_size = 4
```

Comment:
Idk if you did this or this was auto-done by GitHub after I just messed with a setting, but either way, YES. Spaces suck.